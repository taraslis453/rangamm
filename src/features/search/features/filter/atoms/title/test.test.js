import React from 'react'
import {render} from '@testing-library/react'
import {Title} from './index'
describe('<Title/>', () => {
  it('renders properly', () => {
    const {container} = render(<Title color='red'>Title</Title>)
    expect(container.firstChild).toMatchSnapshot()
  })
})
