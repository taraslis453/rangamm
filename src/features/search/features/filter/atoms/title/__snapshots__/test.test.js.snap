// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title/> renders properly 1`] = `
<div
  class="sc-gKsewC hTScYX"
  color="red"
  font-size="1"
>
  Title
</div>
`;
